src/DockAreaTitleBar.cpp

```diff
@@ -285,14 +287,32 @@ CDockAreaTabBar* CDockAreaTitleBar::tabBar() const
 	return d->TabBar;
 }
 
-
 //============================================================================
 void CDockAreaTitleBar::markTabsMenuOutdated()
 {
+	qDebug() << "markTabsMenuOutdated";
```

Comment:
Can you remove this qDebug? I have many messages coming from ADS now in my app that I wish I could toggle off...

Reply:
done